interview-questions.md

```diff
@@ -7,56 +7,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: 
+    An instance variable is created within a class and will be created for each object you make of that class. It only has local scope tied to its particular class, unlike normal variables who's scope depends on where you put them.
 
   Researched answer:
-
+    An instance variable always starts with an '@'. They are private by default, and they need no declaration. They can be used by multiple class methods. Instance variables can be populated with local variables from the arguments list.  
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: 
+    A block is a section of code used as a conditional? Not 100% sure. Researching!
 
   Researched answer:
-
+    A block in Ruby is a section of code, either contained in {} or bookended by a `do` and an `end`. It directly follows a method call and is used as the method argument? Still still super solid on blocks. 
 
```

Comment:
You can think of a block like the code that gets passed to a higher-order function. It has its own scope/local variables, etc. 

JS  ----  variable.map(  < code here >   )   
Ruby  ----  variable.map do     < block code here >    end